Runtime/BacktraceClient.cs

```diff
@@ -838,12 +846,12 @@ private bool ShouldSendReport(BacktraceReport report)
         /// if inner exception exists, client should send report twice - one with current exception, one with inner exception
         /// </summary>
         /// <param name="report">current report</param>
-        private void HandleInnerException(BacktraceReport report, Action<BacktraceResult> callback)
+        private void HandleInnerException(BacktraceReport report)
```

Comment:
Yes - it is recursive, because HandleInnerException reuses this flow but with an inner exception. Callback didn't add anything useful, but added useless allocations and this is why I decided to remove it.